vars/prepareAndRunAnsiblePlaybook.groovy

```diff
@@ -1,7 +1,8 @@
 def call(Map params) {
+  def prepareHardwareConfig = params.config
   def playbook = params.playbook
-  def prepareConfig = params.config
-
+  def vm_role = params.vm_role
+  def vmWareConfig
```

Comment:
We can call `getVMwareConfig` directly in `ansiblePlaybook` step, and remove this global variable.

Reply:
I placed this function in the stage of preparing environment for running ansible, because i think that whole environment should be prepared for running ansible in this stage and stage "run ansible" should be only responsible for executing playbook.

Reply:
Seems reasonable then